tests/org.eclipse.n4js.json.xpect.ui.tests/src/org/eclipse/n4js/json/xpect/ui/tests/JSONXpectPluginTest.java

```diff
@@ -29,6 +29,6 @@
 @RunWith(XpectRunner.class)
 @XpectTestFiles(relativeTo = FileRoot.PROJECT, baseDir = "xpect/nonui", fileExtensions = { "xt" })
 @XpectImport({ JSONSuppressIssuesSetup.class })
-public class JSONXpectTest {
+public class JSONXpectPluginTest {
```

Comment:
why did you change all tests to be plugin tests, even those that do not require this?